terrawrap/models/config_mover.py

```diff
@@ -0,0 +1,271 @@
+# pylint:disable=C0114,C0301
+import os
+from functools import lru_cache
+from os.path import relpath
+from pathlib import Path
+from typing import List, Union, Iterable, Any
+
+import boto3
+import git
+from botocore.exceptions import ClientError
+import hcl2
+
+from terrawrap.utils.config import find_variable_files, parse_variable_files
+from terrawrap.utils.terminal_colors import TerminalColors as Colors
+
+
+class ConfigMover:
+    """
+    Class for moving terraform configuration between directories.
+    This is done by moving .tfstate file in appropriate S3 bucket.
+    """
+
+    tf_state_file_extension = ".tfstate"
+
+    def __init__(
+        self,
+        source_directory: Path,
+        target_directory: Path,
+        tf_config_repo_root: str = "terraform-config",
```

Comment:
```suggestion
        tf_config_repo_root: str = "terraform-config",
```

It's not possible to provide by cli arguments
Is ti possible to use `os.getcwd()` in the handler as the argument to the ConfigMover instance?

Reply:
Ya, this shouldn't be hardcoded. Perhaps you want to use [get_git_root](https://github.com/amplify-education/terrawrap/blob/main/terrawrap/utils/git_utils.py#L38) or [calc_repo_path](https://github.com/amplify-education/terrawrap/blob/main/terrawrap/utils/path.py#L98)